Update module

fmdl opened the pull request:
This update add :
- multi account import
- use IBAN in 'account_number' (for France Only)

fmdl committed:
Update module

This update add :
- multi account import
- use IBAN in 'account_number' (for France Only)

alexis-via commented:
I see that you developped a method _RIBwithoutkey2IBAN(). FYI, for the same need, I use the module "account_bank_statement_import_usability" available in https://github.com/akretion/odoo-usability/tree/8.0/account_bank_statement_import_usability which solves the same problem (match IBAN account number in Odoo for a file which doesn't contain the IBAN) but for all file formats (I have the same issue with OFX for French banks).

Please leave a comment on this PR once you have signed the OCA CLA.

gurneyalex commented:
travis build of 8.0 was recently greenified. can you rebase?

fmdl commented:
Hi, I'am new on github, I don't understand how is work, if a make a mistake please be cool.

@gurneyalex what is "rebase" ?

Have a good day.

fmdl committed:
Update module

- add multi account
- add IBAN Support
- add find partner ID with balise -U- in wire bank transfert
- unique ID renforcement

legalsylvain commented:
you have to run locally

```
git pull --rebase "OCA_remote_name" 8.0
git push --force "IguanaYachts_remote_name" patch-1
```

legalsylvain commented on account_bank_statement_import_fr_cfonb/account_bank_statement_import_fr_cfonb.py:
```diff
@@ -50,6 +52,28 @@ def _parse_cfonb_amount(self, amount_str, nb_of_dec):
+    def _RIBwithoutkey2IBAN(self,banque, guichet, compte):
+        # http://fr.wikipedia.org/wiki/Clé_RIB#V.C3.A9rifier_un_RIB_avec_une_formule_Excel
+        # calcul key rib :
+        lettres = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```
could you change vars name, setting english words ?
